module/library/categories/helper.js

```diff
@@ -173,6 +174,78 @@ module.exports = class LibraryCategoriesHelper {
 					})
 				}
 
+				let projectTemplates = result[0].data
+				let filePathsArray = []
+				for (let project of projectTemplates) {
```

Comment:
@borkarsaish65 can we try this
if front end requet 10 project-templates
fetch them
collect all the category ids from all ten templates
make the array unique
get evidences of categories using unique array of ids
find downloadable url for them in a single call
map back the evidence details to project template data based on the category ids present in them

Reply:
changed the logic now